modules/op-installing-pipelines-operator-in-web-console.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// */openshift_pipelines/installing-pipelines.adoc
+[id="op-installing-pipelines-operator-in-web-console_{context}"]
+= Installing the {pipelines-title} Operator in Web Console
+
+You can install {pipelines-title} using the Operator listed in the {product-name} OperatorHub. When you install the {pipelines-title} Operator, the custom resources required for the Pipelines configuration are automatically installed along with the Operator.
+
+[discrete]
+== Procedure
+
+. In the *Administrator* perspective of the Web Console, navigate to *Operators->OperatorHub*.
+
+. Use the *Filter by keyword* box to search for {pipelines-title} Operator in the catalog. Click on the *{pipelines-title} Operator* tile.
++
+image::op-installed-tile.png[]
+
+. Select the Operator, and click *Continue*.
+
+. Read the brief description about the Operator on the *{pipelines-title} Operator* page. Click *Install*.
++
+image::op-install-subscription.png[]
+
+. On the *Create Operator Subscription* page:
+
+  .. Select *All namespaces on the cluster (default)* for the *Installation Mode*. This mode installs Operator in the default `openshift-operators` namespace, which enables the Operator to watch over, and be made available to all namespaces in the cluster.
+
+  .. Select *Automatic* for  *Approval Strategy*. This ensures that the future upgrades to the Operator are handled automatically by the the Operator Lifecycle Manager (OLM). If you select *Manual*; when a new version of the Operator is available, you will need to manually approve the OLM update request to update to the new version.
+  .. Select an *Update Channel*.
+    *** The 4.4 channel enables installation of the latest stable release of the {pipelines-title} Operator.
```

Comment:
the name of the channel should be `ocp-4.4`

Reply:
done